README.md

```diff
@@ -7,6 +7,118 @@ height="42" />
 
 Runtime for Newfold WP modules and plugins
 
+## Module Responsibilities
+
+* Container is loaded from the brand plugin in Runtime class and it returns a `NewfoldRuntime` object. 
+* This object can be imported in any React components as specified in `Usage` section below. 
+* Primary respnsibility of `wp-module-runtime` is to return `NewfoldRuntime` object which provides WordPress site meta data to React components of any newfold-labs modules in below format.
+
+
+```
+  {
+    "site": {
+        "url": "Your wordpress site url",
+        "title": "Your wordpress site name"
+    },
+    "adminUrl": "Your wordpress site wp-admin url",
+    "base_url": "Your wordpress site index.php url",
```

Comment:
This should ultimately be removed as it doesn't follow the naming convention and shouldn't be in use. I think it is fine to document the values as they do currently exist, but we should strive to remove this.

Reply:
As mentioned in https://github.com/newfold-labs/wp-module-runtime/issues/5 

- `admin_url` was renamed to `adminUrl`
- `base_url` was renamed to `homeUrl`
- `site.url` was renamed to `siteUrl`

But older values were kept for `backward compatibility` so as to avoid breaking Ecommerce module as `src/sdk/NewfoldRuntime.js` file in Ecommerce still uses older values.

Will create JIRA ticket to remove the same in backlog

Reply:
Please review [JIRA](https://jira.newfold.com/browse/PRESS0-1193)

Reply:
Please review [JIRA](https://jira.newfold.com/browse/PRESS0-1193)

Reply:
@wpscholar 
Please review [JIRA](https://jira.newfold.com/browse/PRESS0-1200) for above change.

As discussed ticket will be parked till all tasks under [JIRA](https://jira.newfold.com/browse/PRESS0-1193) are completed.